### ISSUE TITLE
Feature Idea inspired by the Metropolix

### ISSUE THREAD

**Issue by rolfjack** [westlicht/performer#297]
I have been considering getting the new Metropolix, but I realized that the performer actually could do more or less the same - and much more - with a few additions. 

Here are some ideas for Performer inspired by the Metropolix sequencer:

1. The new Metropolix has an “accumulate” feature, where a step rises in tone by some interval each time the step is activated - within  a given range. Fx. 1 tone per step and after 5 steps it returns to normal. This is an interesting feature that could be implanted in the note section of the sequencer. There is already a note range property that could serve as the range for the accumulate feature.

2. The performer currently have a ratched feature that repeats a number of times within the same step. But what if you changed this feature to a “pulse count” like the Metropolix and then added a “gate type” property:
•	Single: A single gate and then a pause for the remaining pulse counts.
•	Multi: One gate for each pulse count.
•	Continuous: A single gate for the full length of the pulse counts.
•	Skip: A pause for the full length of pulse counts.
•	Ratched: The normal ratched feature with a normal length gate but several quick gates.

This would open up the creation of some interesting short sequences, and potentially some longer ones too. The Metropolix is limited to 2 tracks and 8 steps. The Performer has 8 tracks and 64 steps and it would open to so many possibilities. Staying within just 8 steps would make the novation Launchpad a very playable interface.

It is likely that other features of the Metropolix could inspire some additional new features

**Comment by ftdBs** [comment]
Yeah! I was about to get metropolix aswell since it seemed so easy to come up with those interrelated musical events on different channels + gate expansion. Accumulation is also great for rule based progressions. It would be awesome to have those features in performer.
